.circleci/config.yml

```diff
@@ -39,7 +39,6 @@ commands:
           paths:
             - vendor/bundle
       - run: |
-          brew install swiftlint
```

Comment:
This was the result of copying and pasting the CI config from JWTDecode.swift which is written in Swift. This library is written in Objective-C.